packages/axios-extension/src/abap/app-index-service.ts

```diff
@@ -34,4 +34,28 @@ export abstract class AppIndexService extends Axios implements Service {
         const response = await this.get('/', { params });
         return JSON.parse(response.data).results as AppIndex;
     }
+
+    /**
+     * Returns if manifest is first supported.
+     *
+     * @param {string} appId - The id of the app.
+     * @returns {Promise<boolean>} - is manifest first supported.
+     */
+    public async getIsManiFirstSupported(appId: string): Promise<boolean> {
+        try {
+            const params = {
+                'id': appId
+            };
+            const response = await this.get('/ui5_app_mani_first_supported', { params });
+            const parseResponseData = JSON.parse(response.data);
+
+            return parseResponseData as boolean;
+        } catch (error) {
+            if (isAxiosError(error)) {
```

Comment:
Wondering if this should just default to `false` if there is an exception, instead of throwing an exception?

Reply:
Fixed

Reply:
Closing for @longieirl since he is ooo